web/src/app/projects/create-edit/create-edit.component.html

```diff
@@ -32,14 +32,16 @@
                         Entered text is too long.
                     </mat-error>
                 </mat-form-field>
-
             </mat-card-content>
 
             <mat-divider></mat-divider>
             <mat-card-actions>
                 <div fxLayout="row" fxLayoutGap="20px" fxLayoutAlign="center center">
-                    <button mat-stroked-button type="button" routerLink="../">Cancel</button>
-                    <button mat-raised-button type="submit" color="accent" [disabled]="projectForm.invalid">Save</button>
+                    <button mat-stroked-button type="button" routerLink="../" color="warn">Cancel</button>
+                    <button mat-raised-button type="submit" color="accent" [disabled]="projectForm.invalid">
+                        <span *ngIf="!isEdit;else other">Create</span>
```

Comment:
I think only this is needed in the condition?

```typescript
*ngIf="!isEdit"
```

Reply:
Good catch!! Removed 👍